test/e2e/common/node/container_probe.go

```diff
@@ -971,6 +971,11 @@ func RunLivenessTest(f *framework.Framework, pod *v1.Pod, expectNumRestarts int,
 		pod, err = podClient.Get(context.TODO(), pod.Name, metav1.GetOptions{})
 		framework.ExpectNoError(err, fmt.Sprintf("getting pod %s", pod.Name))
 		restartCount := podutil.GetExistingContainerStatus(pod.Status.ContainerStatuses, containerName).RestartCount
+		// Since this is one of the slowest tests, its good to have logs between the 3 minute convergence time...
+		// see https://github.com/kubernetes/kubernetes/issues/110839
```

Comment:
@pohly didn't you fix the problem of slow test not showing output?

Reply:
Ginkgo should start showing where a test is stuck after 5 minutes (see [`--ginkgo.poll-progress-after`](https://github.com/kubernetes/kubernetes/blob/bb2101bac12748aa77adfa15b07b5b8d8dcd0672/hack/ginkgo-e2e.sh#L206).

This seems to be about node e2e, right? Does that run tests with `ginkgo-e2e.sh`? If not, then `--ginkgo.poll-progress-after` may have to be added to the e2e_node invocation.

Reply:
@jayunit100 are you running ginkgo directly instead of using the repo scripts?